examples/real-world/actions/index.js

```diff
@@ -3,10 +3,9 @@ import { CALL_API, Schemas } from '../middleware/api';
 export const USER_REQUEST = 'USER_REQUEST';
 export const USER_SUCCESS = 'USER_SUCCESS';
 export const USER_FAILURE = 'USER_FAILURE';
-/**
- * Fetches a single user from Github API.
- * Relies on the custom API middleware defined in ../middleware/api.js.
- */
+
+// Fetches a single user from Github API.
```

Comment:
It does, just wanted the return variable and parameters passed as well.  Wasn't sure if you were really putting in JSDoc comments.  I can change back and add the missing parts.